cpp/cmake/CPM.cmake

```diff
@@ -0,0 +1,19 @@
+set(CPM_DOWNLOAD_VERSION 0.27.4)
```

Comment:
In other repos we've used a git commit hash to define this instead of the git tag just to make sure it doesn't change out from underneath us.